js/logo.js

```diff
@@ -4345,6 +4345,22 @@ function Logo () {
                 that._setListener(turtle, listenerName, __listener);
             }
             break;
+		case 'noise':
+			if (args[0] == null) {
+                that.errorMsg(NOINPUTERRORMSG, blk);
+                childFlow = args[1];
+                childFlowCount = 1;
+			} else if (args[0] == "white" || args[0] == "pink" || args[0] == "brown") {
+				var noiseSynth = new Tone.NoiseSynth().toMaster();
+				noiseSynth.set("noise.type", args[0]);
+				noiseSynth.triggerAttackRelease("2");
+				childFlow = args[1];
+				childFlowCount = 1;
+			} else {
+				that.errorMsg("Not A Valid Input");
```

Comment:
There is a constant defined near the top of logo.js for invalid inputs. Please use that.